src/components/FlyoutMenu/FlyoutMenu.test.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { fireEvent, render, waitFor, waitForElement } from '@testing-library/react';
+
+import FlyoutMenu, { FlyoutMenuDirection } from './FlyoutMenu';
+
+const applyButtonText = 'Apply';
+const cancelButtonText = 'Cancel';
+
+describe('FlyoutMenu', () => {
+  it('Renders an open transactional menu', async () => {
+    const { queryByText } = render(
+      <FlyoutMenu defaultOpen transactional triggerId="flyout-test" />
+    );
+
+    const applyButtonTest = await waitForElement(() => queryByText(applyButtonText));
+
+    const cancelButtonTest = await waitForElement(() => queryByText(cancelButtonText));
+
+    expect(applyButtonTest).toBeTruthy();
+    expect(cancelButtonTest).toBeTruthy();
+  });
+
+  it('Renders an open menu', async () => {
```

Comment:
This test will easily give us a false positive. I.e. we can remove the `defaultOpen` and the test would still pass... which is proven by the test below. I think it would be better if you added some actual test content to the FlyoutMenu and tested that it was showing/not showing in these open/closed tests.